README.md

```diff
@@ -45,3 +47,16 @@ conn   = train.connection
 result = conn.run_command("apt upgrade -y")
 conn.close
 ```
+
+### Using the InSpec CLI
+
```

Comment:
# install of awsssm-plugin
# setup ( if needed ) for aws creds
# setup ( if needed ) for aws ssm agent on host?

Reply:
Let's make sure we round out the workflow to ensure if we were starting from a clean install to get to working